build/cqs.tsv

```diff
@@ -5,16 +5,23 @@ com.adobe.xmp:xmpcore	5.1.2	compile
 com.beust:jcommander	1.48	compile
 com.clearspring.analytics:stream	2.9.2	compile
 com.drewnoakes:metadata-extractor	2.8.1	compile
+com.esotericsoftware:kryo	3.0.3	compile
 com.esotericsoftware.kryo:kryo	2.21	compile
+com.esotericsoftware:minlog	1.3.0	compile
 com.esotericsoftware.minlog:minlog	1.2	compile
+com.esotericsoftware:reflectasm	1.10.1	compile
 com.esotericsoftware.reflectasm:reflectasm	shaded:1.07	compile
+com.googlecode.concurrent-trees:concurrent-trees	2.6.0	compile
+com.googlecode.cqengine:cqengine	2.7.1	compile
 com.googlecode.efficient-java-matrix-library:core	0.26	compile
 com.google.code.gson:gson	2.3.1	compile
 com.googlecode.json-simple:json-simple	1.1	compile
 com.googlecode.juniversalchardet:juniversalchardet	1.0.3	compile
 com.google.guava:guava	14.0.1	compile
 com.google.guava:guava	17.0	compile
 com.google.guava:guava	18.0	compile
+com.google.protobuf:protobuf-java	2.5.0	compile
+com.google.protobuf:protobuf-java	2.6.1	compile
```

Comment:
yeah, I wondered about that too. protobuf-java 2.6.1 is a cqengine 2.7.1 dependency. I don't know where protobuf-java 2.6.1 is coming from.